app/commands/git/sync_track.rb

```diff
@@ -0,0 +1,46 @@
+module Git
+  class SyncTrack
+    include Mandate
+    initialize_with :track
+
+    def call
+      lookup_head_and_current_commit
+      update_track! if track_config_modified?
+    end
+
+    private
+    attr_reader :current_commit, :head_commit
+
+    def lookup_head_and_current_commit
+      track.git.send(:repo).update!
```

Comment:
We shouldn't use `send()` in production code. So it's probably best to just make `update!` public.

Reply:
> We shouldn't use send() in production code. 

To clarify, we shouldn't use it to access private methods. There are other good reasons to actually use it :)